x/vesting/keeper/integration_test.go

```diff
@@ -36,7 +38,7 @@ import (
 // 23/02 Lock ends
 var _ = Describe("Clawback Vesting Accounts", Ordered, func() {
 	// Monthly vesting period
-	stakeDenom := stakingtypes.DefaultParams().BondDenom
```

Comment:
Why is this change necessary?

Reply:
so, basically, we have to add validator before InitChain() in app test, this is new in sdk v46. The denom is aevmos, so we move to claimstypes.DefaultParams().ClaimsDenom which returns aevmos. stakingtypes.DefaultParams().BondDenom returns denom stake.